src/views/evaluation/Evaluation.js

```diff
@@ -0,0 +1,122 @@
+    </ul>`
+}
+
+const logoButton = function(option = {}){
+    return `
+    <li class="header-elements">
+        <img class= "main-logo" src="${option}">
+    </li>`
+}
+
+const buttonNav = function(option={}, active="" , specialPlacement = ""){
+    return `
+    <li class="header-elements ${specialPlacement}">
+        <a class = "header-elements-link header-elements-linkT ${active} " href="${option.Link}"> ${option.Text}</a>
+    </li>
+    `
+}
+
+
+const constructNav = function(option = {}){
+    const currentPage = "Evaluationv2.html", result =[], specialPlacement = "Loginv2.html";
+    for(var i=0;i<option.Link.length;i++){
+        if(option.Link[i] === currentPage){
+            result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "active-page"))
+        }else{ 
+            if(option.Link[i] === specialPlacement){
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}, "", "right-button"))
+            }else{
+                result.push(buttonNav({Link: option.Link[i], Text: option.Text[i]}))
+            }
+        }
+        
+    }
+    return result.join('')
+}
+
+const tableRows = function (option = {}){
+    return `
+    <tr>
+        <td></td>
+        <td>${option.Nume}</td>
+        <td>${option.Technology}</td>
+        <td>${option.Nivel}</td>
+        <td class="detail-button">Detail</td>
+        <td>
+            <img src="${option.ImageLink}" alt="More Deatails">
+        </td>
+    </tr>`
+}
+
+const insertHeader = function(option = {}){
+    const result = []
+    for(var i=0;i<option.length;i++){
+        result.push(`<th>${option[i]}</th>`)
+    }
+    return result.join('')
+}
+
+const tableHeader = function(option = {}){
+    console.log(option)
+    return `
+    <colgroup>
+        <col class="table-button">
+        <col span="4" >
+        <col class="table-button">
+    </colgroup>
+    <tr>
+        <th></th>
+        ${insertHeader(option)}
+        <th> </th>
+        <th> </th>
+    </tr>`
+}
+
+const tableConstructor = function (option = {}){
+    const result = []
+    result.push(tableHeader(option.tHeader))
+    for (var i=0;i<option.Nume.length;i++){
+         result.push(tableRows({Nume: option.Nume[i], Technology: option.Technology[i], Nivel: option.Nivel[i], ImageLink: option.ImageLink} ))
+    }
+    return result.join('')
+}
+
+const tableBody = function(option = {}){
+    return `
+    <table cellspacing="0" cellpadding= "0" class = "tableBlock" >
+        ${tableConstructor(option)}
+    </table>
+    `
+}
+
+const Footer = function(option = {}){
+    return `	
+    <footer class="footer footerT" >
+        <p>${option.Footer}</p>
+    </footer>`
+}
+const EvaluationPage = function(option ={}){
+    return `${NAV(option.hContent)}${tableBody(option.tContent)}${Footer(option.fContent)}`
+}
+
+window.onload = function(){
+    const result = [];
+    tableContent = {Nume : ["Popescu Adrian", "Dragan Roxana" , "Florescu Mihai", "Gheorghe Andrei"], 
```

Comment:
I find this structure rather strange, as it forces you to always keep all of these inner arrays in sync, with respect to length / properties etc. What if you had 10 elements to start with? And you needed to add another one on the seventh position? See how that might easily lead to trouble? To my mind, it makes more sense to have a single array with multiple objects inside, each object representing the data item to be displayed in a row, complete with name, tech, level (btw, romglish much? :stuck_out_tongue:) and anything else you might need.